website/docs/r/spring_cloud.html.markdown

```diff
@@ -0,0 +1,69 @@
+---
+subcategory: "App Platform"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spring_cloud"
+description: |-
+  Manages an Azure Spring Cloud Service.
+---
+
+# azurerm_spring_cloud
+
+Manages an Azure Spring Cloud Service.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "Southeast Asia"
+}
+
+resource "azurerm_spring_cloud" "example" {
+  name                = "sc-example"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+
+  tags = {
+    Env = "staging"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Service resource. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal. Changing this forces a new resource to be created.
+
+* `location` - (Required) The GEO location of the resource. Changing this forces a new resource to be created. Until November 20, 2019 4:20 PM, only `East US`, `Southeast Asia`, `West Europe`, `West US 2` following locations are supported.
+
+* `tags` - (Optional) Tags of the service which is a list of key value pairs that describe the resource. Changing this forces a new resource to be created.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The id of the Spring Cloud Service.
+
+* `version` - The version of the Spring Cloud Service.

```

Comment:
this can be removed from the documentation since this field isn't worth exposing at this point in time